tests/Unit/Http/Action/Security/LogInActionTest.php

```diff
@@ -109,10 +110,29 @@ public function rendersLogInFormIfAuthenticationFailed()
         $this->assertSame($content, $response->getContent());
     }
 
+    /**
+     * Data provider for redirectsToDashboardIfAuthenticationSucceeded
+     *
+     * @return array
+     */
+    public function userProvider(): array
+    {
+        return [ // admin check, reviewer check, expected redirect
+            [false, false, 'dashboard'],
```

Comment:
For sake of completeness, should we also provide data for another combination

* `[true, true, 'admin']`

?

Reply:
Nope; see below.